tests/network/bond/test_l2_bridge_over_bond.py

```diff
@@ -7,6 +7,7 @@
 import pytest
 
 import utilities.network
+from tests.network.libs import cloudinit as netcloud
```

Comment:
just a thought , can we import NetworkData, EthernetDevice, cloudinit here

Reply:
It is possible, however that will loose the context. E.g. when we see NetworkData called, we miss the context of it originating from cloudinit.

Reply:
these are tests for now limited code base so i thought of going with direct imports can be helpful but i agree totally to what you said. Thanks